app/public/templates/scene.html

```diff
@@ -81,11 +82,22 @@
         <a-entity
           id="primaryCamera"
           camera
+          fov="80"
```

Comment:
Curiosity: any particular reason to change the field of view? Also is 90 default?

Reply:
I am not changing the field of view, I am just making it visible :) 80 is the default for aframe: https://aframe.io/docs/1.1.0/components/camera.html#properties